frontend/src/routes/(frontend)/[accountId]/[statusId]/index.tsx

```diff
@@ -4,20 +4,25 @@ import Status from '~/components/Status'
 import { formatDateTime } from '~/utils/dateTime'
 import { formatRoundedNumber } from '~/utils/numbers'
 import * as statusAPI from 'wildebeest/functions/api/v1/statuses/[id]'
+import * as contextAPI from 'wildebeest/functions/api/v1/statuses/[id]/context'
 import { Link, loader$ } from '@builder.io/qwik-city'
 import StickyHeader from '~/components/StickyHeader/StickyHeader'
 import { Avatar } from '~/components/avatar'
 
 export const statusLoader = loader$<
 	{ DATABASE: D1Database; domain: string },
 	Promise<{ status: MastodonStatus; context: StatusContext }>
->(async ({ redirect, platform, params }) => {
-	const response = await statusAPI.handleRequest(platform.DATABASE, params.statusId)
-	const results = await response.text()
-	if (!results) {
+>(async ({ request, redirect, platform, params }) => {
+	const domain = new URL(request.url).hostname
+	const statusResponse = await statusAPI.handleRequest(platform.DATABASE, params.statusId, domain)
+	const statusText = await statusResponse.text()
+	if (!statusText) {
 		throw redirect(303, '/not-found')
 	}
-	return { status: JSON.parse(results), context: { ancestors: [], descendants: [] } }
+	const contextResponse = await contextAPI.handleRequest(domain, platform.DATABASE, params.statusId)
+	const contextText = await contextResponse.text()
+	const context = JSON.parse(contextText) as StatusContext
```

Comment:
I wonder what we should do if the context is not fetched successfully, any idea?

Reply:
I would throw an error because that shouldn't happen

Reply:
I think we can safely write an error to the logs and respond with `{ancestors: [], descendants: []}`

Reply:
LOL - or what @xtuc said.

Reply:
I guess I can just throw an error, so at least in the browser the user would get a failed network request and it will be slightly more evident that something went wrong

Reply:
I've put there a check for the content and a simple error throw, I thought of try-catching the other calls but it started to look overkill so I went with this very simple solution, let me know if that's ok 🙂